apis/ec2/v1alpha1/generator-config.yaml

```diff
@@ -0,0 +1,57 @@
+ignore:
+  resource_names:
```

Comment:
@Dkaykay I think either this file is not being honored or you need to delete and re-run generation as all EC2 AWS resources appear to be added in the `v1alpha1` group here.

Reply:
should be fixed